assets/templates/_Postcard.hbs

```diff
@@ -1,8 +1,8 @@
 <article class="postcard" onclick="goToDetail({{@index}})">
     {{postBand}}
 
-    {{#ifCond locked true}}
-        {{postLocked}}
```

Comment:
add back postLocked conditional - to represent locked state in template (notice the locked bar at the top of post detail)

Reply:
Done.